web3/stintjson.nim

```diff
@@ -30,3 +23,90 @@ proc fromJson*(n: JsonNode, argName: string, result: var ref UInt256) =
   new result
   result[] = hexStr.parse(StUint[256], 16) # TODO: Handle errors
 
+proc bytesFromJson(n: JsonNode, argName: string, result: var openarray[byte]) =
```

Comment:
Was the json serialization code and the ethcalls singnatures file copied from nimbus (or the former nim-json-rpc repo)? I guess it makes sense to have it shared between the projects, because it describes the same RPC interface.